lalrpop-internals/Cargo.toml

```diff
@@ -0,0 +1,52 @@
+[package]
+name = "lalrpop-internals"
+description = "Internal lalrpop structures.  If you want a parser generator, you should use the lalrpop crate"
+readme = "README.md"
+workspace = ".."
+# This crate is versioned independently - which is the whole point.  We want to
+# be able to bump semver here without bumping lalrpop/lalrpop-util
+version = "1.0.0"
```

Comment:
Should we just set this to "0.1.0" then?

Reply:
I don't have strong feelings one way or the other.  Why prefer 0.1.0 over 1.0.0?

Reply:
It's fair that in terms of version updates, they will act similar. I guess socially, one expects that something that is 1.0 will have less breaking changes/is more settled. I'm not sure if that's true for these internals?